source/MaterialXRenderGlsl/TextureBaker.h

```diff
@@ -42,25 +44,28 @@ class TextureBaker : public GlslRenderer
     }
 
     /// Bake textures for all graph inputs of the given shader reference.
-    void bakeShaderInputs(ShaderRefPtr shaderRef, GenContext& context, const FilePath& outputFolder);
+    void bakeShaderInputs(ConstShaderRefPtr rf, GenContext& context, const FilePath& outputFolder, const string udim = "");
```

Comment:
Maybe just keep the name of this first argument as shaderRef for consistency with the rest?